yaksh/models.py

```diff
@@ -80,6 +81,17 @@
     ("exact", "Case Sensitive"),
     )
 
+legend_display_types = {
+        "mcq": {"category": "objective", "label": "Objective Type"},
```

Comment:
Nested dict not required.
Simply keep ```{"mcq": "Objective Type", ...}``` since name of the dict is enough to understand the intention.

Reply:
This is my opinion is cleaner and more configurable because the decision for the category name and label name for the same category can be made in one place instead of hard coding/if-else structure in the templates.

Reply:
Yes but this ```category``` key is not used anywhere

Reply:
@prathamesh920 removed the category key.